images/runtime/python/cgmanifest.json

```diff
@@ -0,0 +1,34 @@
+{
+    "Registrations": [
```

Comment:
@vijaysaayi may you elaborate on the following:

- Why is this governance file needed?
- Where will `cgmanifest.json` be ingested?
-  If `cgmanifest.json` is ingested outside of Oryx, may we move this file out of Oryx?
- Should `DevelopmentDependency` be `true` since "The vizplugin should be used with viztracer" per: https://github.com/gaogaotiantian/vizplugins

Reply:
@william-msft : 
1. We must [register Open source components to use them in Microsoft Products](https://docs.opensource.microsoft.com/using/ ).
For stacks such as Python, the Component Governance injected into devops pipeline would not automatically pick the open source components. Hence We need to add cgmanifest.json file to explicitly register them.
2. [cgmanifest](https://docs.opensource.microsoft.com/tools/cg/features/cgmanifest/) file can be placed anywhere in the repository and Component Governance of DevOps will pick this file to register the components. 
3. Component Governance is injected DevOps pipeline of Oryx . Hence the cgmanifest file should be placed in Oryx only.
4. [DevelopmentDependency](https://docs.opensource.microsoft.com/tools/cg/features/cgmanifest/) indicates that these packages are used for development only. In our case vizplugin is used in production. Hence I set it to false.

Reply:
Got it, thank you @vijaysaayi!